packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -2309,8 +2309,8 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
       yield put(A.cvvStatusLoading())
       yield call(api.updateCardCvv, payload)
       yield put(A.cvvStatusSuccess())
-    } catch (e) {
-      yield put(A.cvvStatusFailure())
+    } catch (error) {
+      yield put(A.cvvStatusFailure(error))
```

Comment:
I would avoid passing `any` error to the action - it backfires when the expected action payload changes. Maybe the following should be our default error handling strategy:

```suggestion
      const errorPayload = isNabuError(error) ? error : 'The code entered is either invalid or expired. Try Again.'
      yield put(A.cvvStatusFailure(errorPayload))
```

Reply:
Makes sense. We can have some other message - we just need something default there since the API request can throw something generic like timeout error and we do not really want to display it to the user.

Reply:
Alternatively, to avoid issues with translation, we might want to put `NabuError | undefined` into the state and allow component to display default error message with translation in case of `undefined`.